airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json

```diff
@@ -27759,7 +27759,7 @@
     "sourceDefinitionId": "79c1aa37-dae3-42ae-b333-d1c105477715",
     "name": "Zendesk Support",
     "dockerRepository": "airbyte/source-zendesk-support",
-    "dockerImageTag": "0.2.29",
+    "dockerImageTag": "0.2.30",
```

Comment:
A new stream is a minor update following the docs here: https://docs.airbyte.com/contributing-to-airbyte/#examples please fix it.